css1024.css

```diff
@@ -0,0 +1,131 @@
+*,
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  color: #555;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  max-width: 1024px;
+}
+
+.small-screen-menu {
+  display: none;
+}
+
+.bigscreen-menu {
+  display: flex;
+  justify-content: space-between;
+  background: #131313;
+  font-size: 13px;
+  font-weight: 500;
+  line-height: 32px;
+  max-height: 32px;
+  color: hsla(0, 0%, 100%, 0.6);
+  padding: 0.15rem;
+}
+
+.bigscreen-menu ul,
+.big-screen-menu-2 ul {
+  display: flex;
+}
+
+.padlist li {
+  padding: 0.15rem;
+}
+
+.bigscreen-menu ul > li {
+  padding-right: 1rem;
+}
+
+.left-marging {
+  margin-left: 1rem;
+}
+
+.big-screen-menu-2 {
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem;
+  font-family: Shentox, sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  color: #aaa;
+  height: 48px;
+  letter-spacing: 0.04em;
+  line-height: 1;
+}
+
+.cover-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: 1fr 1fr;
+  background-position: center;
+  background-repeat: no-repeat;
+  grid-gap: 5px;
+  margin: 0 auto;
+}
+
+.cover-margin-top {
+  margin-top: -6rem;
+}
+
+.image1 {
+  grid-area: 1 / 1 / 3 / 3;
+  background-position: center;
+  padding: 0;
+  height: 100%;
+  width: 100%;
+}
+
+.image2 {
+  height: 300px;
```

Comment:
- [ ] For the bigger than 1024px screens this height should change to accommodate the growth of image1.